library/src/test/resources/robolectric.properties

```diff
@@ -0,0 +1 @@
+sdk=28
```

Comment:
Should this be 29? `compileSdkVersion` and `targetSdkVersion` are both set to that.

Reply:
This has to be `28` to run on Java 8. If it's `29`, it forces use of Java 9, which currently requires manual configuration in Android Studio to run - see https://stackoverflow.com/questions/56821193/does-robolectric-require-java-9/59649406#59649406.